src/custom/pages/Claim/ClaimNav.tsx

```diff
@@ -28,13 +28,19 @@ export default function ClaimNav({ account, handleChangeAccount }: ClaimNavProps
         <ClaimAccountButtons>
           {!!account && account !== activeClaimAccount && (
             <ButtonSecondary disabled={isAttempting} onClick={() => setActiveClaimAccount(account)}>
-              Your claims
+              Change account to connected wallet
```

Comment:
probably a bit too big. But I guess when Michel review responsive, we might move these and re-label them

Reply:
changed it again but let's review outside this pr